sdk/schemaregistry/schema-registry-avro/test/public/withMessagingClients.spec.ts

```diff
@@ -172,7 +181,9 @@ describe("With messaging clients", function () {
       });
 
       afterEach(async function () {
-        await removeSchema(schemaName);
+        schemaList.push(schemaName);
+        await removeSchemas(schemaList, pipeline, httpClient);
+        schemaList = [];
```

Comment:
Is this necessary?

Reply:
ah I see, the removeSchemas doesn't clear the list afterward, perhaps we can update it to do so?